src/html.js

```diff
@@ -1,45 +1,48 @@
-import React from "react"
-import PropTypes from "prop-types"
-
-const BUILD_TIME = new Date().getTime()
-
-export default class HTML extends React.Component {
-  static propTypes = {
-    body: PropTypes.string,
-  }
-
-  render() {
-    let css
-    if (process.env.NODE_ENV === "production") {
-      css = (
-        <style
-          dangerouslySetInnerHTML={{
-            __html: require("!raw!../public/styles.css"),
-          }}
-        />
-      )
-    }
+import React from 'react'
+import PropTypes from 'prop-types'
 
+const renderInlineCSS = () => {
+  if (process.env.NODE_ENV === 'production') {
     return (
-      <html lang="en">
-        <head>
-          <meta charSet="utf-8" />
-          <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
-          <meta
-            name="viewport"
-            content="width=device-width, initial-scale=1.0"
-          />
-          {this.props.headComponents}
-          {css}
-        </head>
-        <body>
-          <div
-            id="___gatsby"
-            dangerouslySetInnerHTML={{ __html: this.props.body }}
-          />
-          {this.props.postBodyComponents}
-        </body>
-      </html>
+      <style
+        dangerouslySetInnerHTML={{
+          __html: require('!raw!../public/styles.css'),
+        }}
+      />
     )
   }
 }
+
+const HTML = ({
+  headComponents,
+  body,
+  postBodyComponents,
+}) => (
+  <html lang="en">
+    <head>
+      <meta charSet="utf-8" />
+      <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+      <meta
+        name="viewport"
```

Comment:
Could you just remove this file altogether? Gatsby has a default html.js file so generally people should just add things to their `<head>` with plugins and react-helmet (with gatsby-plugin-react-helmet) https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/cache-dir/default-html.js